pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2738,9 +2738,12 @@ self: super: {
   # https://github.com/NixOS/nixpkgs/pull/220972#issuecomment-1484017192
   ffmpeg-light = super.ffmpeg-light.override { ffmpeg = pkgs.ffmpeg_4; };
 
-  # posix-api has had broken tests since 2020 (until at least 2023-01-11)
-  # raehik has a fix pending: https://github.com/andrewthad/posix-api/pull/14
-  posix-api = dontCheck super.posix-api;
+  posix-api = super.posix-api.overrideScope (lself: lsuper: {
+    primitive-unlifted = lself.primitive-unlifted_2_1_0_0;
+
+    # need new run-st because old requires primitive-unlifted <0.2
+    run-st = lself.run-st_0_1_3_2;
+  });
```

Comment:
There is no reason why this should absolutely be blocked from merging, but one problem with this is that anything that depends on `posix-api`, and transitively through another path on `primitive-unlifted` or `run-st` will need to know to add overrides for `primitive-unlifted` and `run-st` as well.

What is it that you're trying to do with `posix-api` where it makes sense to unbreak it like this?

One could argue that it is always better to have more packages compiling (even it if requires messing with transitive dependencies), but I'm wondering how helpful this will be to end-users.

Reply:
Okay, let's go ahead and close this, and we can reopen if someone ends up wanting to use `posix-api`.